pkg/controllermanager/apis/config/types.go

```diff
@@ -30,14 +30,12 @@ type ControllerManagerConfiguration struct {
 	// ClientConnection specifies the kubeconfig file and client connection
 	// settings for the proxy server to use when communicating with the gardener-apiserver.
 	ClientConnection apimachineryconfig.ClientConnectionConfiguration
-	// GardenerClientConnection specifies the kubeconfig file and client connection
-	// settings for the garden-apiserver.
-	// +optional
-	GardenerClientConnection *apimachineryconfig.ClientConnectionConfiguration
```

Comment:
Why you have removed that? The use case is that you might talk directly to the `gardener-apiserver` without going through the `kube-aggregator`

Reply:
Yes we know, however this has never ever been used in practice - also, in general, the API server should rather appear as one entity other than as multiple, separated ones. I think after this long time of maintaining this without any helpful outcome, it is now the time to remove this.

Reply:
@mvladev any follow up? Or OK to remove it?

Reply:
@mvladev any comments?

Reply:
I'm file with removal so /lgtm